docs/content/docs/guide/metrics.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Running the PipelineRun
```

Comment:
what about moving to install (which itsef should be renamed to admin i think)


because that access more a system admin thing

Reply:
Done 
Moved to install folder